include/nbl/builtin/hlsl/workgroup/fft.hlsl

```diff
@@ -0,0 +1,111 @@
+#ifndef _NBL_BUILTIN_HLSL_WORKGROUP_FFT_INCLUDED_
+#define _NBL_BUILTIN_HLSL_WORKGROUP_FFT_INCLUDED_
+
+#include <nbl/builtin/hlsl/subgroup/fft.hlsl>
+#include <nbl/builtin/hlsl/bit.hlsl>
+
+namespace nbl 
+{
+namespace hlsl
+{
+namespace workgroup
+{
+
+template<typename Accessor, typename Scalar, uint32_t WorkgroupSize>
+void store(NBL_REF_ARG(Accessor) sharedmemAccessor, uint32_t threadID, NBL_CONST_REF_ARG(complex_t<Scalar>) lo, NBL_CONST_REF_ARG(complex_t<Scalar>) hi) {
+    sharedmemAccessor.set(threadID + 0 * WorkgroupSize, lo.real());
+    sharedmemAccessor.set(threadID + 1 * WorkgroupSize, hi.real());
+    sharedmemAccessor.set(threadID + 2 * WorkgroupSize, lo.imag());  
+    sharedmemAccessor.set(threadID + 3 * WorkgroupSize, hi.imag());    
+}
+
+template<typename Accessor, typename Scalar, uint32_t WorkgroupSize>
+void load(NBL_CONST_REF_ARG(Accessor) sharedmemAccessor, uint32_t threadID, NBL_REF_ARG(complex_t<Scalar>) lo, NBL_REF_ARG(complex_t<Scalar>) hi) {
+    lo.real(sharedmemAccessor.get(threadID + 0 * WorkgroupSize));
+    hi.real(sharedmemAccessor.get(threadID + 1 * WorkgroupSize));
+    lo.imag(sharedmemAccessor.get(threadID + 2 * WorkgroupSize));  
+    hi.imag(sharedmemAccessor.get(threadID + 3 * WorkgroupSize));    
+}
```

Comment:
Use this little baby:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/include/nbl/builtin/hlsl/memory_accessor.hlsl

Shove the original Accessor inside and you're done.

Reply:
Also having free floating functions just called `store` and `load` is super meh